server/executor/trace_poller.go

```diff
@@ -201,7 +202,7 @@ func (tp tracePoller) donePollingTraces(job tracePollReq, trace traces.Trace) bo
 func (tp tracePoller) handleTraceDBError(job tracePollReq, err error) {
 	run := job.run
 	if errors.Is(err, tracedb.ErrTraceNotFound) {
-		if time.Since(run.CompletedAt) < tp.maxWaitTimeForTrace {
+		if time.Since(run.CreatedAt) < tp.maxWaitTimeForTrace {
```

Comment:
This feels weird. This time now includes the time elapsed since the trigger started, so we need to either change the config name, or change the checked field. I'd incline for the latter, so we can have different timeouts for execution and trace polling